api/pages/saleshomepage.html

```diff
@@ -70,7 +71,7 @@ <h4 class="nav navbar-nav navbar-center">Sales Homepage</h4>
             <div class="card-header">
                 <h4 class="card-title text-center">Announcements</h4>
             </div>
-            <div class="card-body">
+            <div class="card-body" id="announcementsSM">
```

Comment:
changing the url from "orders.html" to "order.html" should fix the conflict on this page